smart_open/s3.py

```diff
@@ -412,6 +412,14 @@ def truncate(self, size=None):
         """Unsupported."""
         raise io.UnsupportedOperation
 
+    def __str__(self):
+        return "s3.SeekableBufferedInputBase(bucket_name='{}', key='{}')".format(
+            self._object.bucket_name, self._object.key)
+
+    def __repr__(self):
+        return "s3.SeekableBufferedInputBase(bucket_name='{}', key='{}')".format(
```

Comment:
I think it makes sense to include the full package name here. It'll be helpful for people outside of smart_open: they'll know what package the class comes from.

```suggestion
        return "smart_open.s3.SeekableBufferedInputBase(bucket_name='{}', key='{}')".format(
```